### DISCUSSION TITLE
奇怪了？为什么你新版我DEV不成功了

### DISCUSSION THREAD

**Discussion by sumingyd** [imsyy/home#90] [Announcements]
GitHub仓库直接拉下来试了。也把发布版本下载下来试了。

奇怪的是。DEV的时候就直接显示首页的加载中。（就是以前刚一打开的时候那个界面，按说下一秒就会进入页面，一直卡在第一屏加载中。）奇了怪了。我哪里操作的不对吗？

powershell给我的警告是：

```sh
 WARN  warnings                                                                                               01:21:54
  Some items were passed to additionalManifestEntries without revisioning info. This is generally NOT safe. Learn more at https://bit.ly/wb-precache.
  - index.html
 ```

一些项目在没有修订信息的情况下被传递到extraManifestEntries。这通常是不安全的。了解更多信息，

我就奇了怪了。。。。

**Comment by sumingyd** [comment]
奇了怪了，用vs code倒是能DEV成功

**Comment by sumingyd** [comment]
不对，我发现直接cmd调试的话  给出来的是3000的端口。用vs给出来的是3001端口。是这里的问题？

**Comment by sumingyd** [comment]
还有个问题，你`public/sociaILinks.json`里写的电报的ICO。地址却挂的推，已经提交了pull，发现把6个全写出来会BUG，ICO一个都不显示了。只能展示5个。

**Comment by imsyy** [comment]
社交链接的问题是因为 pwa 有缓存，你清空一下站点数据，或者在页面停留一段时间等待 pwa 更新完成就可以了

**Comment by sumingyd** [comment]
> 社交链接的问题是因为 pwa 有缓存，你清空一下站点数据，或者在页面停留一段时间等待 pwa 更新完成就可以了

我也终于发现为什么编译之后部署上之后无法访问了。用http端口无法访问。用https才能正常访问。不过我看你说明写的是去掉了强制https。。。。去掉了之后反而强制https了
